TheMovieProject/Cells/UpcomingCollectionViewCell.swift

```diff
@@ -8,8 +8,9 @@
 import UIKit
 
 class UpcomingCollectionViewCell: UICollectionViewCell {
-    
+
     @IBOutlet weak var MovieDuration: UILabel!
```

Comment:
variable declaration için UpperCamelCase yerine (lower) camelCase daha  güzel olur diye düşünüyorum. İlk harfler küçük olacak şekilde. 
Bilmediğim bir isimlendirme varsa konuşalım

Reply:
Katılıyorum, bence lower kullanalım.